modules/social_features/social_group/modules/social_group_flexible_group/social_group_flexible_group.install

```diff
@@ -397,3 +398,22 @@ function social_group_flexible_group_update_8912() {
   // Output logged messages to related channel of update execution.
   return $updateHelper->logger()->output();
 }
+
+/**
+ * Install display view modes for group contents for Flexible group type.
+ */
+function social_group_flexible_group_update_8913(&$sandbox) {
+  // Install new configs.
+  $config_path = drupal_get_path('module', 'social_group_flexible_group') . '/config/install';
```

Comment:
For update hooks its better to load data from a static folder instead of the /config/install, as the content inside that can change before your update hook fires and you don't know the outcome. Keeping that consistent would be great! :D

Reply:
As I understand here we always install new configs, so even if they will be changed later it should be correct to the instal folder, so we don't need to duplicate it in static. Or are there other reasons?

Reply:
@nechai please use file copies from the `config/static` folder in update hooks. You can check this PR as an example: #2277